continuous_integration/hdfs/startup_hdfs.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+HOSTDIR=$(pwd)
+INIT_MARKER=$HOSTDIR/hdfs-initialized-indicator
+
+# Remove initialization marker
+rm -f $INIT_MARKER
+
+# Start the hdfs service
+echo "Starting Docker Container..."
+CONTAINER_ID=$(docker run -d -p 8020:8020 -p 50070:50070 -v $(pwd):/working daskdev/dask-hdfs-testing)
```

Comment:
If the intent is to run tests within the container, not sure exposing the ports is required.